packages/web-components/fast-foundation/src/divider/divider.template.ts

```diff
@@ -1,13 +1,18 @@
 import { ElementViewTemplate, html } from "@microsoft/fast-element";
 import type { FASTDivider } from "./divider.js";
+import { DividerRole } from "./divider.options.js";
 
 /**
  * The template for the {@link @microsoft/fast-foundation#FASTDivider} component.
  * @public
  */
 export function dividerTemplate<T extends FASTDivider>(): ElementViewTemplate<T> {
     return html<T>`
-        <template role="${x => x.role}" aria-orientation="${x => x.orientation}">
+        <template
+            role="${x => x.role}"
+            aria-orientation="${x =>
+                x.role === DividerRole.separator ? x.orientation : void 0}"
```

Comment:
I think the intent would be clear if you if checked for presentation role instead of separator, since that is the exclusion case.

Reply:
Good call, I'll update